style.css

```
@@ -0,0 +1,110 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.header{
```

```suggestion
.header {
```
이렇게 한칸 띄워주시고, 일관된 포맷팅 규칙 적용해주세요 :)